.pre-commit-config.yaml

```diff
@@ -86,5 +86,10 @@ repos:
       language: script
       pass_filenames: false
       entry: "./ci/scripts/run_rat_local.sh"
+    - id: check-cgo-adbc-header
+      name: Ensure CGO adbc.h is sync'd
+      language: script
+      pass_filenames: false
+      entry: "./ci/scripts/run_cgo_drivermgr_check.sh"
```

Comment:
Maybe include a [`files`](https://pre-commit.com/#hooks-files) filter so this only runs when necessary?

You could go a step further and have it pass filenames, then the script could check each of the given files against its copy in `go/adbc/drivermgr` (that might be a bit too fancy though).

Reply:
i missed that, i was trying to find it haha. nice :) i'll update with that filter

Reply:
updated using the files filter, let me know what you think.